--- network.subr.orig	2019-01-06 23:13:49.382184000 +0100
+++ network.subr	2019-01-06 23:13:43.162190000 +0100
@@ -222,6 +222,7 @@
 	if wpaif $1; then
 		/etc/rc.d/wpa_supplicant start $1
 		_cfg=0		# XXX: not sure this should count
+		/etc/rc.d/wpa_cli start $1
 	elif hostapif $1; then
 		/etc/rc.d/hostapd start $1
 		_cfg=0
@@ -252,6 +253,7 @@
 	_cfg=1
 
 	if wpaif $1; then
+		/etc/rc.d/wpa_cli stop $1
 		/etc/rc.d/wpa_supplicant stop $1
 		_cfg=0
 	elif hostapif $1; then
